### PR TITLE
Fix #1360 Tutorial to Participate in Challenge

### LINKED ISSUE #1360: Tutorial to participate in a challenge.
**Description:** Write a tutorial showing how to participate in a challenge on EvalAI. Stepwise instructions are expected.

**Hint:** Go to [https://evalai.cloudcv.org](https://evalai.cloudcv.org) and participate in a challenge.

### PR DESCRIPTION
**Pull request by gautamjajoo**
Fixed #1360 . Added to Docs folder.
@deshraj @aka-jain @RishabhJain2018

### CODE REVIEW

**Comment by aka-jain** [comment]
@RishabhJain2018 please review this.

File: docs/source/participate.md [hunk 1]
```diff
@@ -0,0 +1,43 @@
+## Tutorial to participate in a challenge in EVALAI
```
Comment (RishabhJain2018) [inline docs/source/participate.md]: @gautamjajoo I have added the review in PR #1359. Please remove that question from there and incorporate those changes in this PR. Also, please let me know if you face any issues.

Comment (gautamjajoo) [reply docs/source/participate.md]: Ok..

**Comment by gautamjajoo** [comment]
@deshraj @RishabhJain2018 Please review.

File: docs/source/participate.md [hunk 2]
```diff
@@ -0,0 +1,39 @@
...
+
+EvalAI is an open-source web platform for organizing and participating in AI challenges. EvalAI = evaluating the state of the art in AI.
+Here are NINE (IX) easy steps to participate in a challenge in AVALAI.
```
Comment (deshraj) [inline docs/source/participate.md]: Remove (IX)

Comment (deshraj) [reply docs/source/participate.md]: Replace `AVALAI` with `EvalAI`

**Comment by deshraj** [comment]
Hi @gautamjajoo, I reviewed this carefully. Can you please do the following things:

1. Replace the content of `participate.md` with this https://raw.githubusercontent.com/Cloud-CV/EvalAI/docs-fix/docs/source/participate.md
2. Replace the `docs/source/index.rst` with this https://raw.githubusercontent.com/Cloud-CV/EvalAI/docs-fix/docs/source/index.rst

you will then understand how it's working. Also, you can check the documentation on your local system using the following commands:

```
cd docs
make html
```
Not that whenever you make any change, you will have to run the `make html` command to see the changes. You can view the documentation created in `docs/build/html/index.html`. Let me know if this doesn't make sense.

**Comment by gautamjajoo** [comment]
@deshraj Updated!

**Comment by gautamjajoo** [comment]
@deshraj Please review this PR!!

**Pull request merged**